extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/devconsole/DevConsoleProcessor.java

```diff
@@ -23,4 +39,60 @@ public DevConsoleTemplateInfoBuildItem collectAdditionalEndpoints(
         return new DevConsoleTemplateInfoBuildItem("additionalEndpointInfo", additionalEndpoint);
     }
 
+    // knownPaths variable contain set of static file resources that are available from classpath
+    @BuildStep(onlyIf = IsDevelopment.class)
+    public DevConsoleTemplateInfoBuildItem collectStaticResourcesInfo(
+            ApplicationArchivesBuildItem applicationArchivesBuildItem) throws Exception {
+
+        Set<String> knownPaths = new HashSet<>();
+        for (ApplicationArchive i : applicationArchivesBuildItem.getAllApplicationArchives()) {
+            Path resource = i.getChildPath(StaticResourcesRecorder.META_INF_RESOURCES);
+            if (resource != null && Files.exists(resource)) {
+                collectKnownPaths(resource, knownPaths);
+            }
+        }
+
+        ClassPathUtils.consumeAsPaths(StaticResourcesRecorder.META_INF_RESOURCES, resource -> {
+            collectKnownPaths(resource, knownPaths);
+        });
+
+        List<String> sortedKnownPaths = new ArrayList<>(knownPaths);
+        Collections.sort(sortedKnownPaths);
```

Comment:
You can sort `knownPaths` in place, you don't need to make a new list.

Reply:
Added this 
```
knownPaths.stream()
                .sorted(Comparator.comparing(String::toString))
                .collect(Collectors.toList());

Reply:
I tried adding this for in place sorting of hashset but did not worked , for sorting Hashset I usually do by converting into a list or treeset .

```
        knownPaths.stream()
                .collect(Collectors.toCollection(TreeSet::new));
```

Reply:
Oh, sorry I didn't realise it was a `Set`. Replace `new HashSet` with `new TreeSet` then, and it will be sorted.